app/main.py

```diff
@@ -186,3 +232,52 @@ def datasets_by_project_id(project_id):
         ).json()
     else:
         abort(404, description="Resource not found")
+
+
+@app.route("/thumbnail/<image_id>", methods=["GET"])
+def thumbnail_by_image_id(image_id, recursive_call=False):
+    bl = Biolucida()
+
+    with biolucida_lock:
+        if not bl.token():
+            authenticate_biolucida()
+
+    url = Config.BIOLUCIDA_ENDPOINT + "/thumbnail/{0}".format(image_id)
+    headers = {
+        'token': bl.token(),
+    }
+    response = requests.request("GET", url, headers=headers)
+    encoded_content = base64.b64encode(response.content)
+    if encoded_content == b'eyJzdGF0dXMiOiJBZG1pbiB1c2VyIGF1dGhlbnRpY2F0aW9uIHJlcXVpcmVkIHRvIHZpZXcvZWRpdCB1c2VyIGluZm8uIFlvdSBtYXkgbmVlZCB0byBsb2cgb3V0IGFuZCBsb2cgYmFjayBpbiB0byByZXZlcmlmeSB5b3VyIGNyZWRlbnRpYWxzLiJ9'\
```

Comment:
Is there a better way to test for a failure? This is not very explicit

Reply:
Probably, I think a comment would suffice to understand what that condition is for.